pkg/apis/proxy/v1alpha1/managedproxyconfiguration_types.go

```diff
@@ -311,6 +311,10 @@ type ManagedProxyConfigurationProxyAgent struct {
 	// `additionalArgs` defines args used in proxy-agent.
 	// +optional
 	AdditionalArgs []string `json:"additionalArgs,omitempty"`
+	// `installNamespace` defines the namespace where the proxy-agent will be installed.
+	// +optional
+	// +default=open-cluster-management-cluster-proxy
+	InstallNamespace string `json:"installNamespace,omitempty"`
```

Comment:
will the existing installed `ManagedClusterAddon` reinstall if we edit this value? or rather, is this a valid use-case? as far as i know i think the installed namespace should be a "static" configuration --- we dont need to edit this on the flight. so a `--install-namespace` flag for addon-manager will be sufficient for customizing the namespace, wdyt?